pkg/operator/vttablet/spec.go

```diff
@@ -49,6 +49,7 @@ type Spec struct {
 	BackupLocation           *planetscalev2.VitessBackupLocation
 	BackupEngine             planetscalev2.VitessBackupEngine
 	Affinity                 *corev1.Affinity
+	ExtraFlags               map[string]string
```

Comment:
If we merge VitessShard.VitessExtraFlags into VttabletSpec.ExtraFlags, we don't need this new field.

Reply:
Adjusted in https://github.com/planetscale/vitess-operator/pull/14/commits/6761ea423399f13542e587676abf9bf4ee58db19.